src/PKSim.CLI.Core/Batch/Mapper/ApplicationProtocolMapper.cs

```diff
@@ -27,10 +27,17 @@ public Protocol MapFrom(ApplicationProtocol batchProtocol)
          var simpleProtocol = _protocolFactory.Create(ProtocolMode.Simple, applicationType).DowncastTo<SimpleProtocol>();
          simpleProtocol.Name = batchProtocol.Name ?? "Protocol";
          simpleProtocol.EndTimeParameter.Value = batchProtocol.EndTime;
+         simpleProtocol.EndTimeParameter.IsDefault = false;
          simpleProtocol.Dose.DisplayUnit = simpleProtocol.Dose.Dimension.Unit(batchProtocol.DoseUnit);
          simpleProtocol.Dose.Value = simpleProtocol.Dose.Dimension.UnitValueToBaseUnitValue(simpleProtocol.Dose.DisplayUnit, batchProtocol.Dose);
+         simpleProtocol.Dose.IsDefault = false;
          simpleProtocol.DosingInterval = DosingIntervals.ById(dosingIntervalId);
 
+         if (!simpleProtocol.IsSingleDosing)
+         {
+            simpleProtocol.EndTimeParameter.IsDefault = false;
+         }
```

Comment:
??? 
Isn't `simpleProtocol.EndTimeParameter.IsDefault = false` anyway because of line 30?

Reply:
No. For non singledosing, end parameter is visible and should not be exported to snapshot (e.g.IsDefault false)
What's the problem?

Reply:
```
simpleProtocol.EndTimeParameter.IsDefault = false;
...
if (!simpleProtocol.IsSingleDosing)
{
    simpleProtocol.EndTimeParameter.IsDefault = false;
}

```
`simpleProtocol.EndTimeParameter.IsDefault` was already set to false, so the if case will not change anything.

Reply:
Oh I see. I'll double-check tomorrow. All this code is a one time thing anyways and will be deleted as soon as we have agreement on batch files

Reply:
Oh I see. I'll double-check tomorrow. All this code is a one time thing anyways and will be deleted as soon as we have agreement on batch files

Reply:
done